### ISSUE TITLE
Train ResNet-50 classifier

### ISSUE THREAD

**Issue by henrymwaka** [henrymwaka/CoffeeSentinel-X#2]
Use FastAI or TensorFlow to train a base image classifier and validate against test set.
